src/main/webapp/WEB-INF/jsp/sign-on-web3.jsp

```diff
@@ -11,7 +11,7 @@
         <div class="divider" style="margin-bottom: 1em;"></div>
         
         <div class="center">
-            <a id="metaMaskButton" class="btn-large waves-effect waves-light deep-orange lighten-5">
+            <a id="connectWeb3Wallet" class="btn-large waves-effect waves-light deep-orange lighten-5">
                 <svg width="172" height="33" style="margin-top: 10px;" viewBox="0 0 172 33" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Change the SVG to match the one used in `sign-on.jsp`

Reply:
@hhio618 The button looks wrong in my browser:

![Screen Shot 2022-06-30 at 9 16 24 AM](https://user-images.githubusercontent.com/15718174/176572083-386e6f16-2d2d-43be-9577-be49db3f897a.png)

Are you seeing the same?